Sources/Apodini/Properties/Properties.swift

```diff
@@ -16,19 +16,37 @@ import Foundation
 public struct Properties: Property {
     internal var elements: [String: Property]
     
+    internal var namingStrategy: ([String]) -> String? = Self.defaultNamingStrategy
```

Comment:
> With multiple @Properties applying a naming strategy for them to use the parent's name would result in naming collisions.
Yes, but naming collisions have been possible before, e.g.
```swift
struct A: DynamicProperty {
    // x is used as the name
    @Parameter var x: String
}

struct B: Handler {
    var a: A
    var b: A
}
```
The user is responsible for avoiding those.

> As far as I can see, using this strategy only really applies to cases where you only have one @Parameter in the Properties construct, is this correct?
No. Currently this is the case in the only use-case, but I designed it so that it provides as much flexibility as possible. I could imagine a _shortest unique postfix_ strategy could be useful too in some cases. This would not be possible with your simplified approach.

I consider this code not to be part of the _standard_ API that one would have to use on a daily basis, thus I think it is ok to make it a bit more complex and give more power to the user.

Reply:
I see. As commented [here](https://github.com/Apodini/Apodini/pull/247#discussion_r599526726) this sounds good to me 👍